lambda_code/elasticbeanstalk/Tags/index.py

```diff
@@ -0,0 +1,32 @@
+from cfn_custom_resource import CloudFormationCustomResource
+
+
+class Tags(CloudFormationCustomResource):
+    def validate(self):
+        self.environmentArn = self.resource_properties.get('EnvironmentArn')
+        self.tags = self.resource_properties.get('Tags')
+
+    @staticmethod
+    def tags_to_update(tags):
+        return list(map(lambda tag: {'Key': tag[0], 'Value': tag[1]}, tags.items()))
+
+    def update_tags(self):
+        client = self.get_boto3_session().client('elasticbeanstalk')
+        client.update_tags_for_resource(
+            ResourceArn=self.environmentArn,
+            TagsToAdd=self.tags_to_update(self.tags)
+        )
+        return {'TagsToUpdate': self.tags_to_update(self.tags)}
```

Comment:
Minor: cache the list-ified tags, so you don't need to do the transform twice (cfr line 17)

Reply:
Does a built-in function exist for this ? 
Otherwise, I don't think the extra code makes up for the reduced transformation time.